Cargo.lock

```diff
@@ -452,63 +452,10 @@ dependencies = [
  "serde",
 ]
 
-[[package]]
-name = "beefy-gadget"
-version = "4.0.0-dev"
-source = "git+https://github.com/paritytech/substrate?branch=master#7c8b89b3a6b603902709fc33fdb10a864b8b4204"
-dependencies = [
- "array-bytes",
- "async-trait",
- "fnv",
- "futures",
- "log",
- "parity-scale-codec",
- "parking_lot 0.12.1",
- "sc-client-api",
- "sc-consensus",
- "sc-keystore",
- "sc-network",
- "sc-network-common",
- "sc-network-gossip",
- "sc-utils",
- "sp-api",
- "sp-application-crypto",
- "sp-arithmetic",
- "sp-beefy",
- "sp-blockchain",
- "sp-consensus",
- "sp-core",
- "sp-keystore",
- "sp-mmr-primitives",
- "sp-runtime",
- "substrate-prometheus-endpoint",
- "thiserror",
- "wasm-timer",
-]
-
-[[package]]
-name = "beefy-gadget-rpc"
-version = "4.0.0-dev"
-source = "git+https://github.com/paritytech/substrate?branch=master#7c8b89b3a6b603902709fc33fdb10a864b8b4204"
-dependencies = [
- "beefy-gadget",
- "futures",
- "jsonrpsee",
- "log",
- "parity-scale-codec",
- "parking_lot 0.12.1",
- "sc-rpc",
- "serde",
- "sp-beefy",
- "sp-core",
- "sp-runtime",
- "thiserror",
-]
-
 [[package]]
 name = "binary-merkle-tree"
 version = "4.0.0-dev"
-source = "git+https://github.com/paritytech/substrate?branch=master#7c8b89b3a6b603902709fc33fdb10a864b8b4204"
+source = "git+https://github.com/davxy/substrate?branch=move-beefy-to-consensus#10dffc0aba17b8037b6ca7ec49ca876037a1179d"
```

Comment:
this should be git+https://github.com/paritytech/substrate

same across whole `Cargo.lock`

might be that the bot will automatically update these to the correct thing once the substrate PR is merged, but doing this is non-standard afaik